src/publish.js

```diff
@@ -66,7 +66,7 @@ have published it.\
       return new Promise((resolve, reject) => {
         this.fork(this.atomNpmPath, versionArgs, (code, stderr, stdout) => {
           stderr ??= '';
-          stdout ??= ''; 
+          stdout ??= '';
```

Comment:
The whitespace here was wrong before, but it's arguably slightly noisy changing this without it being in a commit that says "fix whitespace" or some such label.